app/[locale]/page.tsx

```diff
@@ -34,6 +34,35 @@ export async function generateMetadata() {
 }
 
 export default function Home() {
+  const t = useTranslations('components.home');
+
+  // const BeOurFriendsCards = [
+  //   {
```

Comment:
If you don`t use a code don`t put it in a comment. 
Delete it, you can always go back and check what this was before with git